extensions/measurement-tracking/src/panels/PanelStudyBrowserTracking/PanelStudyBrowserTracking.tsx

```diff
@@ -55,7 +57,8 @@ function PanelStudyBrowserTracking({
   const [{ activeViewportId, viewports, isHangingProtocolLayout }, viewportGridService] =
     useViewportGrid();
   const [trackedMeasurements, sendTrackedMeasurementsEvent] = useTrackedMeasurements();
-  const [activeTabName, setActiveTabName] = useState('all');
+
+  const [activeTabName, setActiveTabName] = useState(appConfig.studyBrowserMode || 'all');
```

Comment:
If you are adding new configuration, would it be possible to read it from the customization service using the new paths that you are trying to setup?  That way customizations like that are in a consistent place, and have a well known mechanism for overriding - eg a mode could set the customization value to a different default mode if desired.

Reply:
Sure